emscripten.py

```diff
@@ -1750,27 +1765,81 @@ def emscript_wasm_backend(infile, settings, outfile, libraries=None, compiler_en
   outfile.close()
 
 
-def build_wasm(temp_files, infile, outfile, settings, DEBUG):
-  with temp_files.get_file('.wb.s') as temp_s:
-    backend_args = create_backend_args_wasm(infile, temp_s, settings)
+def wasm_dis(wasm_file, wast_file):
```

Comment:
this should probably be in `tools/shared.py`, we have a bunch of utility functions there like `llvm_opt` etc. that call the various llvm tools, so this could fit alongside them